src/components/video/Video.stories.tsx

```diff
@@ -0,0 +1,30 @@
+import React from 'react'
+import { storiesOf } from '@storybook/react'
+// import { select } from '@storybook/addon-knobs'
+import logo from '../../assets/logo.png'
+import { StyledFigure, PlaceholderButton } from './Video.styles'
+import Video from './Video'
+
+const vidId = 'liJVSwOiiwg?autoplay=1'
```

Comment:
Might be nice to use a Pride in London YouTube video here 🌈 we have a [YouTube channel](https://www.youtube.com/user/LondonLGBTPride)

Reply:
You also don't need to include the `?autoplay=1` as this is already applied in the component itself.